insights/parsers/tests/test_whoopsie.py

```diff
@@ -0,0 +1,45 @@
+import doctest
+import pytest
+
+from insights.parsers import whoopsie
+from insights.parsers.whoopsie import Whoopsie
+from insights.tests import context_wrap
+
+BOTH_MATCHED = """
+/var/crash/.reports-1000-user/whoopsie-report
+""".strip()
+
+NOT_FIND_MATCHED = """
+/usr/bin/find: ‘/var/crash’: No such file or directory
```

Comment:
The test errors must be caused by that the single quotes `‘` in these lines are not ASCII characters.